### DISCUSSION TITLE
New camera settings from QRCode

### DISCUSSION THREAD

**Discussion by Churky23** [gopro/labs#1115] [Q&A]
Hi,
I have a question. After I have created a new camera mode/settings from the App or from the github site and I've scanned it with the camera, where is it stored ? On the gopro I continue to see the previous modes that I've setted on the camera itself not a new mode. What I'm missing? 
Many thanks for your support

**Comment by dnewman-gpsw** [comment]
I don't know what you are expecting. QR Code work like using the menu.  Share the QR code you tried

**Reply by Churky23** [reply]
Hi,
thanks for your reply. 
I make an example. I made this configuration: 
https://gopro.github.io/labs/control/set/?cmd=mV0r1080p60e1!NfHthS1aAwAi64M1x0ag0v0q1dLoV4B4D2R0C5S1W0L3*TUSB=1 .
What I want to know is where this configuration is written in the camera after I have read this with my camera. Does it overwrite the current selection for the video mode or it create a new one? Is It a temporary configuration? If in the future I need to recall this video mode, I can find it on my camera or do I have to scan again from QR code? 
Many thanks

**Reply by dnewman-gpsw** [reply]
Every step is like using the camera menus, and when you setting something on the camera it doesn't forget.

mV0 <- mode video
r1080p60 <- set the current preset (you aren't creating a new one) to 1920x1080 at 60fps
e1 <- HS on
And so on.

So your default video preset have your settings.

**Reply by Churky23** [reply]
Hi,
probably I didn't well explain my doubts but OK, with your answer and some trials I finally understand the point.
If I choose a "preset mode" in the configuration panel (on Github or in the App) the video settings generated are applied to this preset mode on the camera. If I don't choose any "preset mode", the settings are applied to the current mode that is selected on the camera at the time I scan the QRCode.
So, for example, If I don't want to change my preset modes on the camera I have to create a new personal mode, select it and scan the QRCode for the new configuration I'll want ot try, made without any selection in the preset mode field. These new settings will be applied to the new Personal mode ad hoc created.

Is it right these assumption?


![CatturaGP](https://github.com/user-attachments/assets/8ed5b4ed-eb44-4521-b7f1-90402ec3eb6b)
